x-pack/test/functional/apps/infra/feature_controls/infrastructure_security.ts

```diff
@@ -72,7 +72,7 @@ export default function ({ getPageObjects, getService }: FtrProviderContext) {
             ensureCurrentUrl: true,
             shouldLoginIfPrompted: false,
           });
-          await testSubjects.existOrFail('~noDataPage');
+          await testSubjects.existOrFail('kbnNoDataPage');
```

Comment:
With this change we'd be testing for a test subject that is an implementation detail of the black box (from the `infra` perspective) `KibanaPageTemplate`. If the `noDataPage` test subject defined by the specific page template is still there, would it make sense to keep using it?

(This applies to the other similar changes to the `infra` tests as well.)

Reply:
Good point! Fixed.